schema.graphql

```diff
@@ -24,6 +24,31 @@ type CollectionEntity @entity {
   updatedAt: DateTime!
   version: Int!
   volume: BigInt!
+  flippers: [FlipperEntity!] @derivedFrom(field: "collection")
+}
+
+type FlipEvent @entity {
+  id: ID!
+  flipper: FlipperEntity!
+  nft: NFTEntity!
+  soldPrice: BigInt
+  soldTo: String
+  sellTimestamp: DateTime
+  boughtPrice: BigInt!
+  profit: BigInt
+}
```

Comment:
- https://github.com/kodadot/loligo/issues/29#issue-1788063888

Reply:
1. afaik we don't have initial and closing prices. that will fit more into a trading system with candles
2. anyway, this is not the info I'm looking to capture, I am looking to capture flip events for collection activity